Eldev

```diff
@@ -0,0 +1,29 @@
+; -*- mode: emacs-lisp; lexical-binding: t; no-byte-compile: t -*-
```

Comment:
Oh nice. I like `eldev`. Using it for some time now and happy.

Reply:
yeah, it's pretty snappy.